[0.009s] Invoking command in '/home/synaaa/object_detection_test/build/yolov8_def': CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/opt/ros/humble /usr/bin/cmake /home/synaaa/object_detection_test/yolov8_def -DCMAKE_INSTALL_PREFIX=/home/synaaa/object_detection_test/install/yolov8_def
[0.018s] -- Found ament_cmake: 1.3.10 (/opt/ros/humble/share/ament_cmake/cmake)
[0.107s] -- Found std_msgs: 4.2.4 (/opt/ros/humble/share/std_msgs/cmake)
[0.124s] -- Found rosidl_generator_c: 3.1.5 (/opt/ros/humble/share/rosidl_generator_c/cmake)
[0.126s] -- Found rosidl_adapter: 3.1.5 (/opt/ros/humble/share/rosidl_adapter/cmake)
[0.130s] -- Found rosidl_generator_cpp: 3.1.5 (/opt/ros/humble/share/rosidl_generator_cpp/cmake)
[0.136s] -- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
[0.145s] -- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
[0.160s] -- Found rosidl_default_generators: 1.2.0 (/opt/ros/humble/share/rosidl_default_generators/cmake)
[0.278s] [31mCMake Error at /opt/ros/humble/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:59 (message):
[0.279s]   execute_process(/usr/bin/python3 -m rosidl_adapter --package-name
[0.279s]   yolov8_def --arguments-file
[0.279s]   /home/synaaa/object_detection_test/build/yolov8_def/rosidl_adapter__arguments__yolov8_def.json
[0.279s]   --output-dir
[0.279s]   /home/synaaa/object_detection_test/build/yolov8_def/rosidl_adapter/yolov8_def
[0.279s]   --output-file
[0.279s]   /home/synaaa/object_detection_test/build/yolov8_def/rosidl_adapter/yolov8_def.idls)
[0.279s]   returned error code 1:
[0.279s] 
[0.279s]   Traceback (most recent call last):
[0.279s] 
[0.279s]     File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
[0.279s]       return _run_code(code, main_globals, None,
[0.279s]     File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
[0.279s]       exec(code, run_globals)
[0.279s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__main__.py", line 19, in <module>
[0.279s]       sys.exit(main())
[0.279s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/main.py", line 53, in main
[0.279s]       abs_idl_file = convert_to_idl(
[0.279s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__init__.py", line 19, in convert_to_idl
[0.279s]       return convert_msg_to_idl(
[0.279s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/msg/__init__.py", line 28, in convert_msg_to_idl
[0.279s]       msg = parse_message_string(package_name, input_file.stem, content)
[0.280s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 544, in parse_message_string
[0.280s]       msg = MessageSpecification(pkg_name, msg_name, fields, constants)
[0.280s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 388, in __init__
[0.280s]       self.base_type = BaseType(
[0.280s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 205, in __init__
[0.280s]       raise InvalidResourceName(
[0.280s] 
[0.280s]   rosidl_adapter.parser.InvalidResourceName: 'bounding_box_result' is an
[0.280s]   invalid message name.  It should have the pattern '^[A-Z][A-Za-z0-9]*$'
[0.280s] 
[0.280s] Call Stack (most recent call first):
[0.280s]   /opt/ros/humble/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:130 (rosidl_adapt_interfaces)
[0.280s]   CMakeLists.txt:23 (rosidl_generate_interfaces)
[0.280s] 
[0.280s] [0m
[0.280s] -- Configuring incomplete, errors occurred!
[0.280s] See also "/home/synaaa/object_detection_test/build/yolov8_def/CMakeFiles/CMakeOutput.log".
[0.284s] Invoked command in '/home/synaaa/object_detection_test/build/yolov8_def' returned '1': CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/opt/ros/humble /usr/bin/cmake /home/synaaa/object_detection_test/yolov8_def -DCMAKE_INSTALL_PREFIX=/home/synaaa/object_detection_test/install/yolov8_def
